services/blockstorage/test/init_test.go

```diff
@@ -0,0 +1,49 @@
+package test
```

Comment:
Add a test here that tests the registration for the gossip handler, it's part of the spec and not tested

Reply:
We have this mock in every harness setup.